components/BottomSheet/src/MDCBottomSheetController.m

```diff
@@ -93,7 +93,11 @@ - (BOOL)accessibilityPerformEscape {
   [self
       dismissViewControllerAnimated:YES
                          completion:^{
-                           [weakSelf.delegate bottomSheetControllerDidDismissBottomSheet:weakSelf];
+                           if ([weakSelf.delegate respondsToSelector:@selector
```

Comment:
Please capture `weakSelf.delegate` into a variable so it doesn't disappear/change between the if and the use.